### PR TITLE
realtime updates to ETC

### PR DESCRIPTION
**Pull request by terrapin47**
Many fixes related to the pilgrim's path questline ( all areas East of Fedimian ).

### CODE REVIEW

File: ETC.tsv [hunk 1]
```diff
@@ -7753,11 +7753,11 @@ ETC_20150317_007753	{memo X}Collect the back moss through Kowak
 ETC_20150317_007754	{memo X}Spray the back moss to transformed tree roots
 ETC_20150317_007755	Release the real transformed tree roots after finding
 ETC_20150317_007756	{memo X}Defeat the monster after modifying the curse with transformation tree roots
-ETC_20150317_007757	Bring the monster to the around treasure chest and kill them
-ETC_20150317_007758	Release the bound souls by altar
+ETC_20150317_007757	Bring nearby monsters to the treasure chest and kill them
```
Comment (Sunaries) [inline ETC.tsv]: Kill -> defeat

File: ETC.tsv [hunk 2]
```diff
@@ -10528,7 +10528,7 @@ ETC_20150401_010529	{memo X}PCSelfDown:10/Notice:NOTICE_Dm_!:Niole root is cut o
 ETC_20150401_010530	{memo X}PCSelfDown:10/Notice:NOTICE_Dm_!:Niole root is cut off!!:5/NPCDead/Func:SCR_FARM47_HERB_ROOT_EVENT
 ETC_20150401_010531	{memo X}IvnItem:FARM47_1_SQ_060_ITEM_1:1:0/DoTimeAction:Drying after the putting the meats of the monsters:3:MAKING:None
 ETC_20150401_010532	Get rid of the monster by hitting it with a flare
-ETC_20150401_010533	Get the fourth key of Mayvern from Gaule altar
+ETC_20150401_010533	Get the fourth key of Maven from Gaule altar
```
Comment (Sunaries) [inline ETC.tsv:10531]: From the ?

File: ETC.tsv [hunk 3]
```diff
@@ -12763,7 +12763,7 @@ ETC_20150714_012764	You have obtained enough purified essences
 ETC_20150714_012765	You have drunk Romuva Sacred Water.{nl}You became invinsible to demons temporarily!
 ETC_20150714_012766	You have placed the Holy Stone{nl}Lure nearby demons to recharge it!
 ETC_20150714_012767	You have inserted a piece
-ETC_20150714_012768	This pillar has already a piece inserted
+ETC_20150714_012768	This pillar already has a piece inserted.
```
Comment (Sunaries) [inline ETC.tsv]: Remove the period

File: ETC.tsv (hunk 2 above)
Comment (terrapin47) [inline ETC.tsv:10531]: Hmm should be fine without.

**Comment by Sunaries** [comment]
Good to merge. :+1:

**Comment by imcgames** [comment]
Thank you :smile:

**Pull request merged**